src/platform/psvita/psp2_ui.cpp

```diff
@@ -209,7 +209,7 @@ void Psp2Ui::ProcessEvents() {
 	sceCtrlPeekBufferPositive(0, &input, 1);
 	keys[Input::Keys::Z] = (input.buttons & SCE_CTRL_CROSS);
 	keys[Input::Keys::X] = (input.buttons & SCE_CTRL_CIRCLE);
-	keys[Input::Keys::N2] = (input.buttons & SCE_CTRL_TRIANGLE);
+	keys[Input::Keys::F] = (input.buttons & SCE_CTRL_TRIANGLE);
```

Comment:
The decision was taken to support yume nikki here.
The 3DS has the advantage to support the "2" key through touch screen. The vita port might need some love.